java/azure-eventhubs/src/main/java/com/microsoft/azure/servicebus/MessageSender.java

```diff
@@ -201,9 +199,10 @@ public String getSendPath()
 		{
 			this.underlyingFactory.scheduleOnReactorThread(this.sendWork);
 		}
-		catch(IOException ioException)
+		catch (IOException ioException)
```

Comment:
>catch (IOException ioException) [](start = 2, length = 31)

I keep seeing this pattern of calling scheduleOnReactorThread, then try/catch the IOException, then complete some future. I suspect you are just trying to guard against the reactor throwing immediately. We should just do all these catch(es) inside scheduleOnReactoThread() call.